_posts/2019-08-11-pre-post-events-sustainable-train-travels.md

```diff
@@ -0,0 +1,152 @@
+| Valence TGV Rhône-Alpes Sud, FR | Thursday  | 18:22     | 22:44   | Say goodbye to the Alps.                                                                  |
+| Barcelona Sants, ES             | Thursday  |           | 22:44   | 🎉 Hurray, we made it.                                                                     |
+
+The NightJet 471 from Berlin rolls directly to Zürich.
+
+
+### From the East
+
+If you are traveling from the East it might be possible to cross routes with folks from the North and join their travel.
+Lucky us there is a direct connection from Budapest to Zürich.
+Let's look at it:
+
+{:.layout-auto}
+|        City         |    Day    | Departure | Arrival |                                          Notes                                          |
+| ------------------- | --------- | --------- | ------- | --------------------------------------------------------------------------------------- |
+| Budapest-Keleti, HU | Wednesday | 20:40     | 08:20   | Jump on the NightJet and relax.                                                         |
+| Vienna, AT          | Wednesday | 23:25     | 08:20   | Same train, has multiple stops in Austria.                                              |
+| Zürich HB, CH       | Thursday  |           | 08:20   | Time to walk a little in Zürich, meet people and maybe hit a shower and day of hacking. |
+
+Note that you can take a direct train from Warsaw or Prague to Vienna.
+
+### From the South
+
+We will suggest two possible options from Italy. Italy basically has one "train backbone" that goes from South to North so you basically choose how many stops you want to do. For more options and ideas visit [https://www.interrail.eu](https://www.interrail.eu).
+
+{:.layout-auto}
+|     City      |    Day    | Departure | Arrival |                                                       Notes                                                        |
+| ------------- | --------- | --------- | ------- | ------------------------------------------------------------------------------------------------------------------ |
+| Rome, IT      | Wednesday | 11:30     | 14:30   | Get a quick "FrecciaRossa" train to Milan.                                                                         |
+| Milan, IT     | Wednesday | 15:05     | 22:30   | Get the "EuroCity" train to Marseille, France.                                                                     |
+| Marseille, FR | Thursday  | 08:00     | 12:30   | You're in Marseille the whole night. Either sleep or meet witht friends for a night of hacking or an afterhour :-) |
+| Barcelona, SP | Friday    |           | 12:30   | You're in Barcelona and you can rest a bit, visit the city and the following day you'll be cool for the Rustfets   |
+
+**Second option**, with one more stop and less idle time:
```

Comment:
If you highlight this as is, you also need to do the same for the **First option**